templates/frontends-pipeline-pr-checks/cypress_e2e_tests.sh

```diff
@@ -17,12 +17,12 @@ docker run \
 docker cp -a . "${TEST_CONT}:/e2e/"
 
 docker exec -i $TEST_CONT sh -c "npm i"
-docker exec -i $TEST_CONT sh -c "npm run test:e2e"
+docker exec -i $TEST_CONT sh -c "npm run ci:cypress-e2e-tests"
 
 RESULT=$?
 
+docker rm -f $TEST_CONT
```

Comment:
I am afraid we can't handle container cleanup in this way, we need to ensure all the `docker rm`  commands always run - for it, you can either use the `trap` command to invoke a cleanup function - again - to ensure they run in case some of the previous shell commands fail, or (in a Jenkins native way), handled in a `post` step where all of them are cleaned up . For the latter, one way to do it is to pass the container names to the `post` section via Groovy variables

This is the reason why the original scripts do it this way: